ResearchKit/Common/ORKAnswerFormat.h

```diff
@@ -744,6 +752,43 @@ ORK_CLASS_AVAILABLE
 
 
 /**
+ The `ORKEligibilityAnswerFormat` class provides a custom boolean control with 
+ a preferred answer.
+ 
+ The eligibility answer format produces an `ORKBooleanQuestionResult` object.
+ */
+ORK_CLASS_AVAILABLE
+@interface ORKEligibilityAnswerFormat : ORKAnswerFormat
+
+/**
+ Returns an initialized eligibility answer format using the specified preferred answer.
+ 
+ @param preferredAnswer         The preferred answer choice to validate eligibility.
+ 
+ @return An initialized eligibility answer format.
+ */
+- (instancetype)initWithPreferredAnswer:(BOOL)preferredAnswer;
```

Comment:
Wouldn't `expectedAnswer` be clearer than `preferredAnswer`? Semantically, the answer is not *preferred*, as there are no alternatives. If you fail, you are not eligible, hence the answer is *expected*.